app/assets/javascripts/registration_checkboxes.js

```diff
@@ -0,0 +1,39 @@
+var e = document.getElementById("student_discount_current_student");
+
+$('#new_student :checkbox').change(function() {
+  var discount = e.options[e.selectedIndex].text;
+  if (discount == "No") {
+    var singleClass = "50"
+    var doubleClass = "90"
+  } else if (discount == "Yes") {
+    var singleClass = "40"
+    var doubleClass = "70"
```

Comment:
'doubleClass' is already defined.<br>Missing semicolon.<br>'doubleClass' is defined but never used.